x-pack/plugins/endpoint/public/embeddables/resolver/store/data/selectors.ts

```diff
@@ -27,7 +27,7 @@ import {
 } from '../../models/indexed_process_tree';
 
 const unit = 100;
-const distanceBetweenNodesInUnits = 1;
+const distanceBetweenNodesInUnits = 3;
```

Comment:
this change seems to break the "pan to" functionality

Reply:
not sure if its relevant, but the 'center' of the map doesn't adjust based on the presence of the sidebar. We probably need some logic that does that